R/sig-vdiagram.R

```diff
@@ -54,19 +50,3 @@ setMethod("vdiagram", signature = c("character", "missing"),
           }
 )
 
-
-#'@aliases vdiagram
-#'@rdname vdiagram
-setMethod("vdiagram", signature = c("list", "missing"), 
-          definition = function(data) {
```

Comment:
Not sure why this signature is toast, it is the dev site. I.E. how I run things locally without output.